storage_file/README.rst

```diff
@@ -0,0 +1,41 @@
+
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+============
+Storage File
+============
+
+
+External file management depending on Storage Backend module.
+
+It include these features:
+* link to any Odoo model/record
+* store metadata like: checksum, mimetype
+
+Use cases (with help of additionnal modules):
+- store pdf (like invoices) on a file server with high SLA
+- access attachments with read/write on prod environment and only read only on dev / testing
+
+Known issues / Roadmap
+======================
+
+* Update README with the last model of README when migration to v11 in OCA
```

Comment:
Should change this known issue.

Reply:
Also add a readme file like this: https://github.com/OCA/storage/tree/15.0/storage_backend/readme.
To maintain OCA standard.